libs/blocks/read-more/read-more.js

```diff
@@ -0,0 +1,21 @@
+import { createTag } from '../../utils/utils.js';
+
+export default function decorate(block) {
+  const container = block.closest('.section');
+  const buttonText = block.querySelector('strong');
+  const button = createTag('button', { class: 'button read-more-button', 'aria-expanded': 'false' });
+  button.textContent = buttonText.textContent;
+  buttonText.parentNode.replaceChild(button, buttonText);
+
+  const recommendedBlock = container.querySelector('.recommended-articles');
+  if (recommendedBlock) {
```

Comment:
Yes, recommendedBlock requires paddingTop. Also, we need paddingBottom on the container if recommededBlock doesn't exist. Moving the logic to `recommended-articles` block may complicate adding paddingBottom to container.
Basically to make space for `read-more` since its `position:absolute`
![Screen Shot 2023-01-05 at 12 17 32 PM](https://user-images.githubusercontent.com/25802201/210841374-4d08906e-31eb-4f6a-9b60-d2aa83a322c8.png)
![Screen Shot 2023-01-05 at 12 17 46 PM](https://user-images.githubusercontent.com/25802201/210841399-7f90155a-8194-402f-b4ad-c6b8325c60a9.png)

Reply:
ok, let's leave it be then.